tests/main/failover/task.yaml

```diff
@@ -43,6 +44,36 @@ execute: |
         truncate -s 0 /tmp/unpack/lib/systemd/systemd
     }
 
+    inject_systemdloop_failure(){
+        cat <<EOF > /tmp/unpack/lib/systemd/system/deadlock.service
+    [Unit]
+    Before=sysinit.target
+    DefaultDependencies=no
+    [Service]
+    Type=oneshot
+    ExecStartPre=-/bin/sh -c "echo 'DEBUG: $(date): deadlocked system' >/dev/console"
+    ExecStartPre=-/bin/sh -c "echo 'DEBUG: $(date): deadlocked system' >/dev/ttyS0"
+    ExecStart=/bin/systemctl start deadlock.service
+    RemainAfterExit=yes
+    [Install]
+    RequiredBy=sysinit.target
+    EOF
+        cat <<EOF > /tmp/unpack/lib/systemd/system/autoreboot.service
+    [Unit]
+    DefaultDependencies=no
+    Description=Force reboot if booting did not finish after 20s (the system does not do it by itself)
```

Comment:
@mvo5 very good point thanks, I've added a comment to make clear the lack of the required capability. Let me know if we should wait for it to be in place to add this test.